cognite/extractorutils/unstable/core/base.py

```diff
@@ -435,6 +437,7 @@ def run(self) -> None:
             with extractor:
                 extractor.run()
         """
+        self._start_time = datetime.now(tz=timezone.utc)
```

Comment:
nice to set it to UTC! Have had a ton of problems correlating when the customer has some funky timezones in use when running on their systems. Very confusing.